buildpacks/ruby/src/layers/bundle_install_layer.rs

```diff
@@ -353,9 +355,14 @@ fn bundle_install(env: &Env) -> Result<(), CommandError> {
         ],
     );
 
-    user::log_info(format!("\nRunning command:\n$ {command}"));
+    user::log_info(format!("\nRunning command:\n$ {command_string}"));
 
-    command.stream()?;
+    command
+        .output_and_write_streams(std::io::stdout(), std::io::stderr())
+        .map_err(|error| cmd::annotate_which_problem(error, bundle.into(), env.get("PATH")))
+        .map_err(cmd::os_command_error)
+        .and_then(|output| cmd::check_non_zero(output, cmd::OutputState::AlreadyStreamed))
```

Comment:
We could provide two functions here for the variants and then you can call them without arguments.

Reply:
Would need two functions:

- `check_non_zero_already_streamed`
- `check_non_zero_include_output_in_error`

It's a bit of a mouthful. Not sure If I like it or not. Being explicit here seems nice. I don't think this is an optimization we need to make right now.